.github/workflows/altinn-k6-lib-release.yml

```
@@ -0,0 +1,39 @@
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: libs/k6
+    if: startsWith(github.ref, format('refs/tags/{0}', altinn-k6-lib-))
```

actionlint (1.7.4)

undefined variable "altinn-k6-lib-". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)